### PR TITLE
Removed $ prefix from all the commands

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Updated commands to remove '$' prefix.
Fixes https://github.com/hirosystems/chainhook/issues/273

### CODE REVIEW

File: docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ events_keys = ["*"]
 
 In this section, you will configure a chainhook to communicate with the network. Run the following command in your terminal and generate the `Chainhook.toml` file.
 
-`$ chainhook config generate --testnet`
+`chainhook config generate --testnet`
```
Comment (MicaiahReid) [inline docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md:49]:
```suggestion
```
chainhook config generate --testnet
```
```
@LakshmiLavanyaKasturi I'm curious on your thoughts for also turning these commands in to code blocks with ```. That adds a "Copy" button to the code block so developers can easily copy each command.

**Comment by MicaiahReid** [comment]
@LakshmiLavanyaKasturi I'm curious on your thoughts for also turning these commands i to code blocks by wrapping them in ``` instead of `. That adds a "Copy" button to the code block so developers can easily copy each command.

**Comment by lgalabru** [comment]
"```console" would be even better!

**Comment by LakshmiLavanyaKasturi** [comment]
@MicaiahReid Pushed changes. Please review. Thanks!

**Review by MicaiahReid — APPROVED** [review]
This looks great, thanks @LakshmiLavanyaKasturi! 🎉

**Pull request merged**